001-cron.php

```diff
@@ -112,6 +112,13 @@ function wpcom_vip_cron_control_event_object_to_string( $event ) {
 }
 
 /**
+ * When Cron Control's Internal Event force-publishes posts that missed their scheduled time, enable async transitions
+ */
+function wpcom_vip_enable_async_actions_for_forced_posts( $pid ) {
+	Automattic\VIP\Async_Publish_Actions\_queue_async_hooks( 'publish', 'future', get_post( $pid ) );
```

Comment:
We should probably have a sanity check here to make sure that the method exists, or switch to firing an action instead.

Reply:
I was probably trying to keep all of the Cron Control stuff together, but this could just move to https://github.com/Automattic/vip-go-mu-plugins/blob/master/async-publish-actions.php.